test/crd_checks.go

```diff
@@ -104,3 +105,59 @@ func WaitForServiceExternalIPState(c *clients, namespace, name string, inState f
 		return inState(r)
 	})
 }
+
+func taskRunSucceed(name string) func(tr *v1alpha1.TaskRun) (bool, error) {
```

Comment:
hm i guess it's not required but i feel like these should be Public and have docstrings :thinking:

Reply:
Fair enough :angel: I did the *smallest* change possible, but I'm definitely +:100: on the docstring :angel: Will update :wink: